pyworkflow/project/project.py

```diff
@@ -380,8 +380,9 @@ def getCurrentProtocolView(self):
         else:
             viewKey = self.getProtocolViews()[0]
             self.settings.setProtocolView(viewKey)
-            print("PROJECT: Warning, protocol view '%s' not found." % currentView)
-            print("         Using '%s' instead." % viewKey)
+            if currentView is not None:
+                print("PROJECT: Warning, protocol view '%s' not found." % currentView)
```

Comment:
This warning is there for a long time ago. It is not related to this PR, but maybe we can check the conditions and get rid of it.

Reply:
I left the warning in case the view disappear. It mainly appear when opening test projects that never had a view at creation.

Reply:
Ok